doc/changelog/02-specification-language/13183-using-att.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  Attribute `#[using="..."]` now supported by Definition and (Co)Fixpoint.
+  It has the same effect of :cmd:`Proof using ...`, which is only available in
```

Comment:
```suggestion
  Attribute :attr:`using` now supported by Definition and (Co)Fixpoint.
  It has the same effect as :cmd:`Proof using`, which is only available in
```

(Note the top part of my suggestion requires adding the documentation of the attribute, with the `.. attr::` directive first; while the bottom part is needed to make the refman compilation succeed.)